ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -121,20 +141,18 @@ object Queries {
       sql"""INSERT INTO ledger_offset VALUES ($party, $tpid, $newOffset)""".update.run
     )
 
-  def insertContracts[F[_]: cats.Foldable: Functor, CA: JsonWriter, WP: JsonWriter](
-      dbcs: F[DBContract[SurrogateTpId, CA, WP]])(implicit log: LogHandler): ConnectionIO[Int] =
-    Update[DBContract[SurrogateTpId, JsValue, JsValue]](
+  @silent // pas is demonstrably used; try taking it out
```

Comment:
Not really sure what warning we are trying to silence.

Reply:
Unused parameter `pas`.